guides/doc-Configuring_DNS_DHCP_TFTP/master.adoc

```diff
@@ -0,0 +1,18 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: admin-network
+:admin-network:
+:ProductName: {ProjectServer}
```

Comment:
Why is this included? Our attributes still give me a headache after almost two years on the project, but I think we're only using `:ProductName:` in `master.adoc`s for example to be able to render it as `Foreman Server` in server installation guides and `Smart Proxy Server` in proxy installation guides. I don't think that's required in your new guide.

I believe that for this guide, dropping line 5 and replacing all occurrences of `{ProductName}` with `{ProjectServer}` should work just fine. Or am I missing something?

Reply:
> Why is this included? Our attributes still give me a headache after almost two years on the project, but I think we're only using `:ProductName:` in `master.adoc`s for example to be able to render it as `Foreman Server` in server installation guides and `Smart Proxy Server` in proxy installation guides. I don't think that's required in your new guide.

Good call. In my review I aimed to consistently always write `{ProjectServer} or {SmartProxyServer}` which removes the need for `{ProductName}`. After we merge this PR we're much closer to removing `ProductName` as an attribute.